era5cli/utils.py

```diff
@@ -108,15 +105,13 @@ def _format_hours(values: list) -> list:
     """
     returnlist = []
     for value in values:
-        assert (int(value) >= 0), (
-            'invalid value specified for hours: {}'.format(value))
-        assert (int(value) <= 23), (
-            'invalid value specified for hours: {}'.format(value))
-        returnlist += ["{}:00".format(str(value).zfill(2))]
+        assert int(value) >= 0, f"invalid value specified for hours: {value}"
+        assert int(value) <= 23, f"invalid value specified for hours: {value}"
+        returnlist += [f"{str(value).zfill(2)}:00"]
     return returnlist
 
 
-def _print_multicolumn(header: str, info: list):
+def print_multicolumn(header: str, info: list):
```

Comment:
Is this the formatter's choice or yours? It's not clear to me why some function names were changed from `_function_name()` to `function_name()` and others not.

Reply:
I changed function definitions where they had the underscore, but were used in different modules. The underscore generally denotes "internal use". `_zpadlist` in `utils`, for example, is only used within utils.py. However, `print_multicolumn` is used outside of utils, and should therefore not have an underscore.